rwgame/debugstate.cpp

```diff
@@ -129,6 +130,53 @@ DebugState::DebugState(RWGame* game, const glm::vec3& vp, const glm::quat& vd)
 	m->addEntry(Menu::lambda("Cull Here", [=] {
 		game->getRenderer()->setCullOverride(true, _debugCam);
 	}, entryHeight));
+	m->addEntry(Menu::lambda("Unsolid garage doors", [=] {
+
+		std::vector<std::string> garageDoorModels {
```

Comment:
you could probably use const static std::array of const char* here to avoid doing memory allocations/for a bit more performance (if it matters)